js/ui/tree_list/ui.tree_list.focus.js

```diff
@@ -92,8 +92,6 @@ core.registerModule('focus', extend(true, {}, focusModule, {
 
                     that.expandAscendants(key).done(function() {
                         dataSource.load({
-                            filter: that.getCombinedFilter(),
-                            sort: that.getController('columns').getSortDataSourceParameters(!dataSource.remoteOperations().sorting),
```

Comment:
Why did you remove filter and sort?

Reply:
The TreeList takes filter and sort later when loading. So I removed this to avoid duplication